script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -207,24 +208,26 @@ She even starts clapping her hands. She's totally just making fun of me.
 [sha:e086f0f496595a2437e62096f68415920b92f423]{
 -- Page 47, Offset 14921.
 -- そのあまりにもとぼけた反応で、俺の頭もようやく落ち着いてくれた。
-I finally start to calm down from seeing how flagrantly she's playing dumb.
+Faced with this flagrant display of feigned ignorance, I finally start to calm down.
 }
 [sha:b05a6e4b64139d03fca01465c4082b3778aec857]{
 -- Page 48, Offset 14922.
 -- 「……もういいです。どうせ俺なんて先輩の中じゃどうでもいい扱いなんだろうし。騒いですみませんでした」
-"Whatever... I guess Senpai just doesn't give a crap about me.
- Sorry for getting so worked up."
+"Fine, whatever...
+ Looks like you doesn't give a crap about me after all.
+ Sorry for bothering you."
 }
 [sha:157bb6b3d11473535b9384179e928cdc159051a9]{
 -- Page 49, Offset 14923.
 -- 「ぁ……いえ、そう言われると、その……
-"Oh, um. When you put it like that, I――
+"Oh, um... When you put it like that, well...
```

Comment:
You really don't have to follow the Japanese punctuation so precisely.

Reply:
there's also no reason to avoid it. I could take or leave the second change but the first half is improved